src/course-updates/data/api.js

```diff
@@ -0,0 +1,85 @@
+import { getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
+export const getCourseUpdatesApiUrl = (courseId) => `${getApiBaseUrl()}/course_info_update/${courseId}/`;
+const updateCourseUpdatesApiUrl = (courseId, updateId) => `${getApiBaseUrl()}/course_info_update/${courseId}/${updateId}`;
+export const getCourseHandoutApiUrl = (courseId) => {
+  const formattedCourseId = courseId.split('course-v1:')[1];
+  return `${getApiBaseUrl()}/xblock/block-v1:${formattedCourseId}+type@course_info+block@handouts`;
+};
+
+/**
+ * Get course updates.
+ * @param {string} courseId
+ * @returns {Promise<Object>}
+ */
+export async function getCourseUpdates(courseId) {
+  const { data } = await getAuthenticatedHttpClient()
+    .get(getCourseUpdatesApiUrl(courseId));
+
+  return data;
+}
+
+/**
+ * Create new course updates.
```

Comment:
please coordinate the naming `updates` and `update` for both doc and function name

Reply:
Done